internal/controlplane/handlers_authz_test.go

```diff
@@ -250,6 +253,7 @@ func TestProjectAuthorizationInterceptor(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
+		tc := tc
```

Comment:
There shouldn't be any need for this in Go 1.22

Reply:
Agreed, but I was getting a bunch of linter complaints.  I can [turn off the `loopclosure` analysis in VSCode](https://github.com/golang/vscode-go/issues/227), but I worry that it might catch other issues.  If you've got a good recipe for ignoring these in our go1.22 code and not elsewhere, let me know.  😁